session/schema_amender_test.go

```diff
@@ -51,8 +51,11 @@ func initTblColIdxID(metaInfo *model.TableInfo) {
 	}
 	for i, idx := range metaInfo.Indices {
 		idx.ID = int64(i + 1)
-		// TODO unique index is not supported now.
-		idx.Unique = false
+		if idx.Name.L == "f_g" {
```

Comment:
what does 'f_g" mean?

Reply:
It's a index in the test schema marked with unique index flag.

Reply:
In the `MockSignedTable` is the test schema.